src/secure_mount.rs

```diff
@@ -111,7 +120,14 @@ pub(crate) fn mount() -> Result<String> {
                         .map(|path| {
                             info!("Changed path {} owner to root.", path);
                         })
-                        .map_err(|_| Error::SecureMount)?;
+                        .map_err(|e| {
+                            Error::SecureMount(
+                                format!(
+                                    "unable to change secure path dir owner to root: received exit code {}",
+                                    e.code().unwrap().unwrap() //#[allow_ci]
```

Comment:
_Generally_ we want to avoid `unwrap` but there may be cases where it's fine. It may be worth noting in a comment why it's in use in these cases.

Reply:
Completely agree. I don't think it can be avoided here, as they are `Option` types (but if you know a way, let me know). I added comments as suggested.